test/controllers/FileChecksControllerSpec.scala

```diff
@@ -224,8 +226,13 @@ class FileChecksControllerSpec extends FrontEndTestHelper with TableDrivenProper
     "call the fileCheckProgress endpoint" in {
       val graphQLConfiguration: GraphQLConfiguration = new GraphQLConfiguration(app.configuration)
       val consignmentService: ConsignmentService = new ConsignmentService(graphQLConfiguration)
-      val controller = new FileChecksController(getAuthorisedSecurityComponents, new GraphQLConfiguration(app.configuration),getValidStandardUserKeycloakConfiguration,
-        consignmentService,frontEndInfoConfiguration)
+      val controller = new FileChecksController(
+        getAuthorisedSecurityComponents,
+        new GraphQLConfiguration(app.configuration),
```

Comment:
Instead of creating a new '`GraphQLConfiguration`' you can use the one declared on line 227